python/tvm/driver/build_module.py

```diff
@@ -275,7 +275,7 @@ def build(
 
     annotated_mods, target_host = Target.check_and_update_host_consist(annotated_mods, target_host)
 
-    rt_mod_host = _driver_ffi.preprocess_module(annotated_mods, target_host)
+    rt_mod_host = _driver_ffi.build(annotated_mods, target_host)
```

Comment:
There are some difference related "overrode the target host" logic between 'preprocess_module' and 'build',  is this change may fix any potential corner case or cause any new corner case issue?